tests/installation/module_registration/register_extensions_and_modules.pm

```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Register Application module
+#          in "Extension and Module Selection" dialog
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+use testapi qw(save_screenshot get_var);
+
+sub run {
+    my @scc_addons = split ',', get_var('SCC_ADDONS');
+    $testapi::distri->get_module_registration()->register_extension_and_modules([@scc_addons]);
+    save_screenshot;
+
+    # when some module (e.g. workstation extension) requires registration, provide separate code
```

Comment:
If scc_addons contain live patch module also need reg code, then the case will failed, is there possible check the scc_addons content and then do separate_registration_code?